wom/src/main/scala/wdl4s/wdl/Scope.scala

```diff
@@ -34,6 +34,8 @@ trait Scope {
     } else throw new UnsupportedOperationException("children is write-once")
   }
 
+  lazy val childGraphNodesSorted: List[WdlGraphNode] = (children.toSet[Scope] collect { case g: WdlGraphNode => g }).toList.sortWith((first, second) => first.isUpstreamFrom(second))
```

Comment:
I hear there's a magic `filterByType` method in Lenthall that could clean this up